stories/tree/tree.stories.js

```diff
@@ -1,897 +1,1019 @@
+                                <span class="fd-tree__text">Level 3</span>
+                            </a>
+                        </div>
+                    </li>
                 </ul>
             </li>
-            <li class="fd-tree__item" role="treeitem">
-                <div class="fd-tree__row">
-                    <div class="fd-tree__col fd-tree__col--control">
-                        Second Level
+            <li role="treeitem" aria-level="2" class="fd-tree__item">
+                <div tabindex="0" class="fd-tree__item-container">
+                    <div class="fd-tree__content">
+                        <span class="fd-tree__text">Level 2</span>
                     </div>
-                    <div class="fd-tree__col">
-                        Secondary Value
+                </div>
+            </li>
+        </ul>
+    </li>
+    <li role="treeitem" aria-level="1" class="fd-tree__item">
+        <div tabindex="0" class="fd-tree__item-container has-highlight-indicator--success is-navigated">
+            <a href="https://sap.github.io/fundamental/" class="fd-tree__content has-navigation-indicator">
+                <span class="fd-tree__text">Level 1</span>
+            </a>
+        </div>
+    </li>
+    <li role="treeitem" aria-level="1" class="fd-tree__item">
+        <div tabindex="0" class="fd-tree__item-container has-highlight-indicator--success">
+        <a href="https://sap.github.io/fundamental/" class="fd-tree__content has-navigation-indicator">
+            <span class="fd-tree__text">Level 1</span>
+        </a>
+        </div>
+    </li>
+</ul>
+`;
+
+treeWithExpandedLevel6AndNavigationLinks.parameters = {
+    docs: {
+        iframeHeight: 1350,
+        storyDescription: `
+When Level 6 is expanded to show up to 12 levels:
+
+Indentations are calculated with 0.25rem
+
+- Level 2 Changes to be indented by 0.25rem
+- Level 3 Changes to be indented by 0.5rem
+- Level 4 Changes to be indented by 0.75rem
+- Level 5 Changes to be indented by 1rem
+- Level 6 Changes to be indented by 1.25rem
+
+0.25rem is added to each indentation up to level 12.
+
+If only a subset of the tree items are navigable (contain links) you should indicate those by displaying a navigation arrow. This is achieved by applying the <code class="docs-code">.has-navigation-indicator</code> class to the <code class="docs-code">.fd-tree\\_\\_content</code> element and <code class="docs-code">.fd-tree\\_\\_item-container--active</code> modifier class to the <code class="docs-code">.fd-tree\\_\\_item-container</code>. 
+
+Do not show arrows if all items are navigable. In this case apply the <code class="docs-code">.fd-tree--active</code> modifier class to the root tree. This will add states (hover, selected, active) to all tree items.
+
+If an item is navigated this can be shown by applying <code class="docs-code">.is-navigated</code> class to the <code class="docs-code">.fd-tree\\_\\_item-container</code> element.
```

Comment:
If an item is "navigated" or "navigable"? Navigable makes more sense here linguistically.

Reply:
Alright! So, the proper way is: "To indicate that an item has been navigated to, apply the <code class="docs-code">.is-navigated</code> class to the <code class="docs-code">.fd-tree\\_\\_item-container</code> element." 😄